### PR TITLE
Update 15-yehjovi.md

### PR DESCRIPTION
**Pull request by yehjovi**

### CODE REVIEW

**Comment by yehjovi** [comment]
test PR

File: _slides/15-yehjovi.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Im here
```
Comment (rogermow) [inline _slides/15-yehjovi.md:6]: Looks good.

**Review by rogermow — APPROVED** [review]

**Comment by rogermow** [comment]
Approved.

**Review by yehjovi — COMMENTED** [review]
good

**Comment by rogermow** [comment]
Merge done.

**Pull request merged**